lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb

```diff
@@ -13,15 +13,13 @@ Make an appointment at the British embassy in Bogota to swear an affidavit (writ
 
 ## Fees
 
-<%= render partial: 'consular_fees_table_items.govspeak.erb',
-           collection: calculator.services,
-           as: :service,
-           locals: { calculator: calculator } %>
-
+Service | Fee
```

Comment:
Sorry @mgrassotti - I only commented on the first erb template but the same needs to happen for the others for the Colombia. Sorry I didn't make that clear before. Do you want me to add a comment in the others as well?

Also I think you can remove the:

```
Service | Fee
- |-
Affidavit or affirmation for marriage | £50
```